runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -85,6 +85,7 @@ public Object getPayload(CorfuRuntime runtime) {
         if (value instanceof LogEntry) {
             if (!Address.isAddress(((LogEntry) value).getGlobalAddress())) {
                 ((LogEntry) value).setGlobalAddress(getGlobalAddress());
+                ((LogEntry) value).setRuntime(runtime);
```

Comment:
Can you just pass the serializer here directly?

Reply:
I thought about it, but then I found that SMREntry calls `serializerType.deserialize(objBuf, runtime)` and JsonSerializer will need runtime to build an object

`rt.getObjectsView().build()
                        .setStreamID(new UUID(b.readLong(), b.readLong()))
                        .setType((Class<? extends ICorfuSMR>) Class.forName(smrClassName))
                        .open();`